dkms.conf

```diff
@@ -1,7 +1,5 @@
 PACKAGE_NAME="rtl8812AU"
-PACKAGE_VERSION="#MODULE_VERSION#"
+PACKAGE_VERSION="4.3.22"
```

Comment:
Why hardcode this version number? As it is now, it takes its value from `include/rtw_version.h` which is now `4.3.22-beta` and without this change I could merge your changes to the other branches/driver versions as well.